### DISCUSSION TITLE
Hue dimmer switch (324131092621) + Hue white ambiance Aurelle (3216231P5) - cannot toggle light temperature with dimmer anymore

### DISCUSSION THREAD

**Discussion by grubyak** [Koenkk/zigbee2mqtt#5672] [Devices]
I have following setup:
- https://www.zigbee2mqtt.io/devices/324131092621.html
- https://www.zigbee2mqtt.io/devices/3216231P5.html

Out of the box dimmer allows to toggle light temperature with consecutive clicks on "turn on" button:
- click: turns on
- click: warm white -> neutral white
- click: neutral white -> daylight white
- click: daylight white -> warm white

After pairing dimmer and lamp to zigbee2mqtt everything works as before, except that consecutive click on "turn on" button on the dimmer do not toggle light temperature anymore. Temperature can still be changed with mqtt message, but not with dimmer itself anymore. Is anyone having similar issue?

Bonus question: Is it possible to change the brightness stepping of Aurelle when using dimmer? Currently it changes brightness with jumps of 30 (within a range 1..254) when using a dimmer buttons. I would like to adjust the stepping to a different value.

**Comment by arendwierks** [comment]
I am interested in this as well.
I know there is a solution for Trädfri remotes ([link](https://www.zigbee2mqtt.io/devices/E1524_E1810.html#binding)), but I hope there is some workarround to make Hue remotes capable of switching scene's too.
